test/scripts/filters/external_link.js

```diff
@@ -21,8 +21,7 @@ describe('External link', () => {
 
     hexo.config.external_link.enable = false;
 
-    const result = typeof externalLink(content);
-    result.should.eql('undefined');
+    should.equal(externalLink(content), undefined);
```

Comment:
```suggestion
    should.not.exist(externalLink(content));
```

chai has `should.not.exist()` syntax.

Reply:
I don't understand the obvious reasons for obscuring null and undefined.
However, I think that should.equal () was not used in previous test code, and I think we should only use should.exist () to avoid confusion.

Reply:
@segayuu `to.be.` should be used with chai's `expect` syntax, not with `should` syntax.